app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```

Comment:
I have edited it now, just has the home page content that i needed and no header or title information for the welcome page. So this page will be yielded in the application html page